app/Http/Controllers/Admin/StatisticsController.php

```diff
@@ -29,7 +29,7 @@ public function index(Request $request)
         $servers = Server::all();
```

Comment:
I'm trying to avoid using any of the model direct eloquent calls in this fashion as they are difficult to unit test properly. There should be a repository call for all of these, its going to be quite a constructor most likely.